Alignment/OfflineValidation/interface/PreparePVTrends.h

```diff
@@ -0,0 +1,303 @@
+#ifndef PREPAREPVTRENDS_H_
```

Comment:
```suggestion
#ifndef ALIGNMENT_OFFLINEVALIDATION_PREPAREPVTRENDS_H_
```
please use full-fledged inclusion guards to minimize chances of collisions.